README.md

```diff
@@ -9,22 +9,22 @@ You can [download Impact][download] by visiting the Impact Website.
 
 ## Help and Support
 
-If you need help using Impact, you should instead ask for it on the [discord] server. Use the `#help` channel so that you don't clutter up `#general` with support questions. Also make sure to check `#announcements`, pinned messages and use the search tool to see if your question has already been answered.
+If you need help using Impact, you should instead ask for it on the [discord] server. Use the `#help` channel so that you don't clutter up `#general` with support questions. Make sure to check `#announcements`, pinned messages and use the search tool to see if your question has already been answered.
 
-It is also recommended to check the wiki first, it may have your fix:
-[Common Issues](https://github.com/ImpactDevelopment/ImpactIssues/wiki/Common-Issues-FAQ)
+It is also recommended checking the wiki first, it may have your fix:
+[Common Issues][Common Issues].
```

Comment:
this is only linked once, no sense in changing it

Reply:
i wanted to keep the same md style, its much cleaner dis way